### PR TITLE
fix: maker with cs fixer

### PR DESCRIPTION
**Pull request by nikophil**
since some times, maker bundle is shipped with cs fixer, so it fixes the cs of created files. This feature cannot be deactivated, so I updated the expected file to match the code style

this feature appeared in maker 1.49.0 so I decided to bump this version (dev req) in order to have the same behavior even with `--prefer-lowest` option

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -49,7 +49,7 @@ ARG UID=1001
 ARG XDEBUG_HOST="172.17.0.1"
 
 RUN addgroup --system --gid ${UID} ${USER} ; \
-    adduser --system --disabled-password --uid ${UID} --ingroup ${USER} ${USER} ; \
+    adduser --system --home /home/${USER} --disabled-password --uid ${UID} --ingroup ${USER} ${USER} ; \
```
Comment (nikophil) [inline docker/Dockerfile:52]: this is needed now the image uses the `bookworm`  version

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ jobs:
           - 27017:27017
 
     env:
-      MYSQL_URL: mysql://root:1234@127.0.0.1:3306/zenstruck_foundry?serverVersion=5.7
+      MYSQL_URL: mysql://root:1234@127.0.0.1:3306/zenstruck_foundry?serverVersion=5.7.42
```
Comment (nikophil) [inline .github/workflows/ci.yml:65]: this fixes a new deprecation in doctrine
https://github.com/doctrine/dbal/blob/3.6.x/src/Driver/AbstractMySQLDriver.php#L58-L63

File: docker/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ARG PHP_VERSION
 
-FROM php:${PHP_VERSION}-cli
+FROM php:${PHP_VERSION}-cli-bullseye
```
Comment (nikophil) [inline docker/Dockerfile:3]: at work we had a lot of subtle problems coming from the new `bookworm` release of debian

foundry's CI was failing as well for unknown reasons, so I preferred to stick to bullseye for now

File: tests/Functional/Bundle/Maker/MakeFactoryTest.php [hunk 4]
```diff
@@ -11,6 +11,8 @@
 
 namespace Zenstruck\Foundry\Tests\Functional\Bundle\Maker;
 
+use Composer\InstalledVersions;
+use Composer\Semver\VersionParser;
```
Comment (ndench) [inline tests/Functional/Bundle/Maker/MakeFactoryTest.php]: Are these required? They don't appear to be used.

**Review by ndench — APPROVED** [review]
Looks great!

File: docker/Dockerfile (hunk 1 above)
Comment (kbond) [reply docker/Dockerfile:52]: > this is needed now the image uses the bookworm version

Below you switched away from bookworm?

File: tests/Functional/Bundle/Maker/MakeFactoryTest.php (hunk 4 above)
Comment (nikophil) [reply tests/Functional/Bundle/Maker/MakeFactoryTest.php]: indeed

File: docker/Dockerfile (hunk 1 above)
Comment (nikophil) [reply docker/Dockerfile:52]: yes, but let's keep it, it will be there if we upgrade at some point

**Pull request merged**